sympy/algebras/tests/test_quaternion.py

```diff
@@ -12,8 +13,8 @@ def test_quaternion_construction():
     assert q + q == Quaternion(2*x, 2*y, 2*z, 2*w)
 
     q2 = Quaternion.from_axis_angle((sqrt(3)/3, sqrt(3)/3, sqrt(3)/3), 2*pi/3)
-    assert q2 == Quaternion(Rational(1/2), Rational(1/2),
-                            Rational(1/2), Rational(1,2))
+    assert q2 == Quaternion(Rational(1,2), Rational(1,2),
```

Comment:
This change is not related to this PR anyway, please don't make this change here.